arrow-libs/fx/arrow-fx-resilience/src/commonMain/kotlin/arrow/fx/resilience/Schedule.kt

```diff
@@ -301,11 +300,11 @@ public sealed class Schedule<Input, Output> {
    * Changes the delay of a resulting [Decision] based on the [Output] and the produced delay.
    *
    */
-  @ExperimentalTime
-  public fun modify(f: suspend (Output, Duration) -> Duration): Schedule<Input, Output> =
-    modifyNanos { output, d -> f(output, d.nanoseconds).toDouble(NANOSECONDS) }
+  public abstract fun modify(f: suspend (Output, Duration) -> Duration): Schedule<Input, Output>
 
-  public abstract fun modifyNanos(f: suspend (Output, Double) -> Double): Schedule<Input, Output>
+  @ExperimentalTime
+  public fun modifyNanos(f: suspend (Output, Double) -> Double): Schedule<Input, Output> =
+    modify { output, d -> f(output, d.toDouble(NANOSECONDS)).nanoseconds }
```

Comment:
We don't need any of the `Nanos functions anymore now that `Duration` is finally stable.

Reply:
I've marked them for deprecation, since they were part of the exposed API